### PR TITLE
Add check for understanding

### PR DESCRIPTION
**Pull request by lizbur10**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,6 +147,16 @@ Later on, we'll also see how to customize the structure of this JSON data even
 further using a **serializer**. For now though, just think of the view as the
 response that's being returned from the controller.
 
+## Check For Understanding
+
+Before you move on, make sure you can answer the following questions:
+
+1. In the MVC architecture, what is the primary responsibility of each of the
+   three componenents?
+2. In the MVC architecture, which component should contain the most logic? Which
```
Comment (ihollander) [inline README.md]: "layer" instead of "component"

Comment (lizbur10) [reply README.md]: Done - thanks @ihollander

**Review by crutell331 — APPROVED** [review]
👍🏾

**Comment by lizbur10** [comment]
Thanks @ihollander and @crutell331 ! I will continue moving forward and continue to tag you for reviews.

**Pull request merged**